sig-scheduling/annual-report-2022.md

```diff
@@ -65,12 +65,12 @@
 
 ## Membership
 
-- Primary slack channel member count:
-- Primary mailing list member count:
-- Primary meeting attendee count (estimated, if needed):
-- Primary meeting participant count (estimated, if needed):
-- Unique reviewers for SIG-owned packages: <!-- in future, this will be generated from OWNERS files referenced from subprojects, expanded with OWNERS_ALIASES files -->
-- Unique approvers for SIG-owned packages: <!-- in future, this will be generated from OWNERS files referenced from subprojects, expanded with OWNERS_ALIASES files -->
+- Primary slack channel member count: 3050
+- Primary mailing list member count: 655
+- Primary meeting attendee count (estimated, if needed): 10
```

Comment:
em.. one thing I think related here is the APAC meeting, usually hosted by @Huang-Wei ❤️
but we have very little attendees each time. 🤷‍♂️ but  I think we can still move on if we have agenda. FYI.

Reply:
The APAC meeting doesn't run smoothly atm. We don't have concrete agenda items and less attendees than expected. TBH I'm considering closing it sometime in the future.

Reply:
Yes, this grows the burden for you can't just ignore it, but another problem is the primary sig scheduling meeting is usually too late for APAC, 12 am or 1 am in the midnight, I admit I have selfishness here. 😄

Reply:
Negotiating a meeting time across the globe is always challenging :(

@alculquicondor @ahg-g what's your take?

Reply:
Maybe we should go back to alternated times, preserving meetings every 2 weeks.

We have 3 TLs, two in PDT and one in EDT. Ideally each meeting should fall within business hours of at least two TLs. The current one falls within business hours of all of us. I think we can keep this one and move the other one, but I think it will still have to be slightly out of business hours.

Reply:
>  I admit I have selfishness here.

Not at all, we should accommodate for as many contributors as possible.

Reply:
> We have 3 TLs, two in PDT and one in EDT. Ideally each meeting should fall within business hours of at least two TLs. The current one falls within business hours of all of us. I think we can keep this one and move the other one, but I think it will still have to be slightly out of business hours.

It would be nice to have this. 😄

Reply:
I guess nothing to add to the report with regards to this thread, please let me know if not

Reply:
See https://github.com/kubernetes/community/issues/7254